binding.cc

```diff
@@ -355,11 +359,12 @@ namespace zmq {
     info.GetReturnValue().Set(info.This());
   }
 
-  bool
+  short
   Socket::IsReady() {
```

Comment:
I would rename this method, as it is no longer a function to `bool`. `PollForEvents` is ugly, but better than the currently misleading name.